components/headers/ExperimentHeader.vue

```diff
@@ -122,16 +122,21 @@ export default class extends Vue {
 
 .experiment-header__media {
   width: 100%;
-  display: grid;
-  grid-template-columns: repeat(2, 1fr);
-  grid-gap: 0.5rem;
+}
+
+@media (min-width: 600px) {
```

Comment:
Across the framework, we are working the other way around (I mean, [with `max-width: 600px`](https://github.com/qiskit-community/community.qiskit.org/blob/master/components/sections/PageSection.vue#L68)). If you don't mind, change it to be consistent with other media queries. If you think this way is better, could you explain why, please?

Reply:
Actually I did it how you suggest in my first commit:
https://github.com/qiskit-community/community.qiskit.org/pull/100/commits/dbc2506219b6f7a3cf0306f4d6c8559863d9eba8
and then I refactored it 😂

The reason I think this approach is better is because:
- We can take advantage of the default CSS style of `display` that is: `block`, so we only modify this style from 600px forward. It's easier to understand it like that than in reverse.

- In all the previous projects I have worked on, they were "mobile-first" so the CSS media-queries was always declared using `min-width`. We should consider it depending on how many users we have from mobile or desktop.

Reply:
I think consistency beats code-reuse in this occassion although I won't block on this. Consider opening a issue for homogenizing our media queries. Perhaps part of qiskit/qiskit.org#272. Figure out.

Reply:
I totally agree consistency beats code-reuse. And yes, I can be part of issue 272!